packages/mui-base/src/useSlider/useSlider.ts

```diff
@@ -550,15 +550,11 @@ export function useSlider(parameters: UseSliderParameters): UseSliderReturnValue
 
   React.useEffect(() => {
     const { current: slider } = sliderRef;
-    slider!.addEventListener('touchstart', handleTouchStart, {
-      passive: doesSupportTouchActionNone(),
-    });
+    slider!.addEventListener('touchstart', handleTouchStart);
```

Comment:
This can remain as it was before. Only the `removeEventListener` function doesn't need the third parameter.